docs/getting-started/full-site-editing.md

```diff
@@ -0,0 +1,42 @@
+# Full Site Editing
+
+Full Site Editing is the umbrella project name for various sub-projects within Gutenberg. Projects under Full Site Editing (FSE) include the Site Editor, Widgets Editor, Navigation Editor, Global Styles, numerous Site/Post/Page specific blocks, Query block, Navigation block, Templates, and block themes.
+
+## Get Involved
+
+The majority of full site editing features are avilable in the Gutenberg plungin, and an ongoing [FSE Outreach program](https://make.wordpress.org/test/handbook/full-site-editing-outreach-experiment/) is running with calls for testing amongst a community of folks.
+
+- Join in on Slack at #fse-outreach-experiment
+- Participate in the [Calls for Testing](https://make.wordpress.org/test/tag/fse-testing-call/) and test and feedback.
+- See the [How to Test FSE instructions](https://make.wordpress.org/test/handbook/full-site-editing-outreach-experiment/how-to-test-fse/) to get setup to test FSE features.
```

Comment:
I think that setting up FSE needs to be a separate part of the documentation outside of the outreach program.

Reply:
Since it is enough for now, to install a theme with support, and to activate Gutenberg, perhaps this can just be added under the first headline  # Full Site Editing.

Reply:
Updated with TT1 blocks in the directory and moved up instructions to jump in